src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -53,7 +53,7 @@ limitations under the License.
 <!--====================show mac address-table address <mac-address> command ======================-->
     <COMMAND
         name="show mac address-table address"
-        help="Show mac address table address for mac-address"
+        help="Show MAC address table address for mac-address"
```

Comment:
remove hyphen from "mac-address" and capitalize MAC there too

Reply:
done